.travis.yml

```diff
@@ -23,6 +26,7 @@ before_install:
   - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/symfony:${SYMFONY_VERSION}"; fi;
```

Comment:
given that we use a single component which itself has no dependency, no need to require the full framework.